scala/README.md

```diff
@@ -382,44 +439,48 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
 
     1. Instead of `c exists (_ == 0)`, use `c contains 0`.
 
-0. Do not import `scala.collection.mutable._` or even a single mutable collection directly. Instead, import the `mutable` package and use it explicitly as a namespace prefix to denote mutability:
+    This is not only a matter of style: each of these recommendations is algorithmically more efficient that the naive approach.
+
+0. Do not import `scala.collection.mutable._` or even any single mutable collection directly. Instead, import the `mutable` package itself and use it explicitly as a namespace prefix to denote mutability:
```

Comment:
The additional plus of this is you avoid name conflicts if you use mutable and immutable structures close together. Not just a subtlety thing!